scss/_patterns_notifications.scss

```diff
@@ -1,62 +1,146 @@
 @import 'settings';
 $notification-content-vert-padding: $spv-inner--small;
 $notification-content-icon-space: 2 * $sph-inner + map-get($icon-sizes, default);
-$notification-icon-vert-offset: 0.5 * (map-get($line-heights, default-text) - map-get($icon-sizes, default));
+$notification-icon-vert-offset: 0.6rem;
+$notification-text-margin-bottom: 0.6rem;
 
 // Notification style patterns
 @mixin vf-p-notification {
   // The mixin for basic notification styling
   %vf-notification {
     @extend %vf-is-bordered;
 
+    background-position: $sph-inner $notification-icon-vert-offset;
+    background-repeat: no-repeat;
+    background-size: map-get($icon-sizes, default);
     border-radius: 0 $border-radius $border-radius 0;
-    display: flex;
     margin-bottom: $spv-outer--scaleable;
-    overflow: hidden;
-    padding: 0;
+    margin-top: -1px;
+    padding-left: $notification-content-icon-space;
+    position: relative;
 
     &.is-borderless {
+      background-color: transparent;
+      background-position-x: 0;
       border-color: transparent;
-
-      .p-notification__content {
-        background-color: transparent;
-        background-position: 0 0.5 * (map-get($line-heights, default-text) - map-get($icon-sizes, default));
-        padding: 0 0 0 2 * $sph-inner;
-      }
+      margin-top: 0;
+      padding: 0 0 0 2 * $sph-inner;
 
       &::before {
         display: none;
       }
     }
 
     &.is-inline {
-      .p-notification__title {
-        display: inline;
+      .p-notification__content {
+        margin-bottom: $notification-text-margin-bottom;
+        padding-top: map-get($nudges, nudge--p) + map-get($browser-rounding-compensations, h5);
       }
 
+      .p-notification__title,
       .p-notification__message {
         display: inline;
+        margin: 0;
+        padding: 0;
+      }
+    }
+
+    &.is-modal {
+      box-shadow: $box-shadow;
```

Comment:
You can do `@extend %vf-has-box-shadow` to make sure it's future-proof and consistent with any changes we may introduce.